diff --git a/selenium/webdriver/common/selenium_manager.py b/selenium/webdriver/common/selenium_manager.py
index 73085fd..7f4ccb7 100644
--- a/selenium/webdriver/common/selenium_manager.py
+++ b/selenium/webdriver/common/selenium_manager.py
@@ -42,7 +42,7 @@ class SeleniumManager:
         """
 
         if os.getenv("SE_MANAGER_PATH"):
-            path = os.getenv("SE_MANAGER_PATH")
+            path = Path(os.getenv("SE_MANAGER_PATH"))
         else:
             platform = sys.platform
 
