diff --git a/pyproject.toml b/pyproject.toml
index ae7d68d..5f3301e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,7 @@
 [build-system]
-requires = ["setuptools", "setuptools-rust"]
+# [conda-forge] avoid build-time rust
+# requires = ["setuptools", "setuptools-rust"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
@@ -43,8 +45,9 @@ exclude = ["test*"]
 namespaces = false
 # include-package-data is `true` by default in pyproject.toml
 
-[[tool.setuptools-rust.bins]]
-target = "selenium.webdriver.common.selenium-manager"
+# [conda-forge] shipped by selenium-manager-feedstock
+# [[tool.setuptools-rust.bins]]
+# target = "selenium.webdriver.common.selenium-manager"
 
 [project.urls]
 Repository = "https://github.com/SeleniumHQ/selenium/"
