diff --git a/py/selenium/webdriver/common/selenium_manager.py b/py/selenium/webdriver/common/selenium_manager.py
index 72d3e46cd6..8e62f07729 100644
--- a/py/selenium/webdriver/common/selenium_manager.py
+++ b/py/selenium/webdriver/common/selenium_manager.py
@@ -44,24 +44,10 @@ class SeleniumManager:
         if os.getenv("SE_MANAGER_PATH"):
             path = os.getenv("SE_MANAGER_PATH")
         else:
-            platform = sys.platform
+            import shutil
+            path = Path(shutil.which("selenium-manager") or shutil.which("selenium-manager.exe"))
 
-            dirs = {
-                "darwin": "macos",
-                "win32": "windows",
-                "cygwin": "windows",
-            }
-
-            directory = dirs.get(platform) if dirs.get(platform) else platform
-            file = "selenium-manager.exe" if directory == "windows" else "selenium-manager"
-
-            path = Path(__file__).parent.joinpath(directory, file)
-
-        if not path.is_file() and os.environ["CONDA_PREFIX"]:
-            # conda has a separate package selenium-manager, installs in bin
-            path = Path(os.path.join(os.environ["CONDA_PREFIX"], "bin", file))
-            logger.debug(f"Conda environment detected, using `{path}`")
-        if not path.is_file():
+        if not Path(path).is_file():
             raise WebDriverException(f"Unable to obtain working Selenium Manager binary; {path}")
 
         logger.debug(f"Selenium Manager binary found at: {path}")
