diff --git a/pyproject.toml b/pyproject.toml
index 4af5df0..6be8d85 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "selenium"
 version = "4.27.0"
-license = "Apache 2.0"
+license.text = "Apache-2.0"
 description = "Official Python bindings for Selenium WebDriver."
 readme = "README.rst"
 requires-python = "~=3.8"
@@ -40,7 +40,7 @@ zip-safe = false
 [tool.setuptools.packages.find]
 include = ["selenium*"]
 exclude = ["test*"]
-namespace = false
+namespaces = false
 # include-package-data is `true` by default in pyproject.toml
 
 [project.urls]
