diff --git a/selenium/webdriver/common/selenium_manager.py b/selenium/webdriver/common/selenium_manager.py
index f174bda..6ba2892 100644
--- a/selenium/webdriver/common/selenium_manager.py
+++ b/selenium/webdriver/common/selenium_manager.py
@@ -41,7 +41,7 @@ class SeleniumManager:
         :Returns: The Selenium Manager executable location
         """
 
-        if path := os.getenv("SE_MANAGER_PATH") is not None:
+        if (path := os.getenv("SE_MANAGER_PATH")) is not None:
             return Path(path)
         else:
             platform = sys.platform
